source/source-connector/fundamentals/specify-schema.txt

```diff
@@ -182,6 +182,21 @@ following options:
    output.format.value=schema
    output.schema.infer.value=true
 
+The source connector can infer schemas for incoming documents that
+contain nested documents stored in arrays. Starting in Version 1.9 of the
+{+connector+}, schema inference does not default to ``string`` type
+assignment if there are differences between nested documents
+described by the following cases:
+
+- A field is present in one document but missing in another.
+- A field is present in one document but ``null`` in another.
+- A field is an array with elements of any type in one document but an
```

Comment:
not just empty document but another document that has a different datatype as well

Reply:
added this point